yarn.lock

```diff
@@ -1626,11 +1626,16 @@
   resolved "https://registry.npmjs.org/@types/node/-/node-12.20.55.tgz#c329cbd434c42164f846b909bd6f85b5537f6240"
   integrity sha512-J8xLz7q2OFulZ2cyGTLE1TbbZcjpno7FaN6zdJNrgAdrJ+DZzh/uFR6YrTb4C+nXakvud8Q4+rbhoIWlYQbUFQ==
 
-"@types/node@^14.14.35", "@types/node@^14.18.34":
+"@types/node@^14.14.35":
```

Comment:
does this line hint at leftovers of node 14 types?

Reply:
This comes from transitive dependency.

Reply:
do you know which dependency? can we upgrade it? I guess it's not ideal to have types from both node versions, not sure if it will bite us but it seems unclean to me.